lib/FundDistribution/FundDistributionFields/FundDistributionFields.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Field,
+} from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  KeyValue,
+  Row,
+  Select,
+  TextField,
+} from '@folio/stripes/components';
+
+import {
+  parseNumber,
+  validateRequired,
+  calculateAmount,
+} from '../../utils';
+
+const FundDistributionFields = ({
+  required, disabled, price, funds, fund, elem, fundCode,
+}) => {
+  return (
+    <Row>
+      <Col xs={4}>
+        <Field
+          component={Select}
```

Comment:
Since you're touching this, we have a common components for select fields in this very repo, please check out